purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -3988,7 +3982,8 @@ class PurchasesTest {
         type: ProductType
     ): List<StoreProduct> {
         val storeProducts = skusSuccessfullyFetched.map { sku ->
-            stubSkuDetails(sku, type.toGoogleProductType()!!).toStoreProduct()
+            if (type == ProductType.SUBS) createStoreProductWithoutOffers(sku)
+            else createMockOneTimeProductDetails(sku).toStoreProduct()
```

Comment:
i wonder if we want to test some of our storeProductConversions now that they have more logic... not saying for this PR, just thinking out loud

Reply:
yes, I plan on adding more tests, I wanted to fix the existing ones for now because that sets up a good base to write new tests. Sorry I should have been more clear in the PR title and body